generators/uscore-r4/templates/module.yml.erb

```diff
@@ -18,9 +18,13 @@ test_sets:
         - ManualRegistrationSequence
         - StandaloneLaunchSequence
         - EHRLaunchSequence
-      - name: US Core R4
+      - name: US Core R4 Patient Based Profiles
         run_all: true
-        sequences:<% sequences.each do |sequence| %>
+        sequences:<% sequences.reject { |seq| seq[:delayed_sequence] }.each do |sequence| %>
```

Comment:
Could this be assigned to a variable so that the logic doesn't live in the template?

Reply:
Moved to variable